manifests/base/odh-dashboard-crd.yaml

```diff
@@ -37,8 +37,6 @@ spec:
                       type: boolean
                     disableISVBadges:
                       type: boolean
-                    disableUserManagement:
```

Comment:
Can you also remove this from backend/src/type.ts, backend/src/utils/constants.ts, frontend/src/type.ts,  and frontend/src/utilities/useWatchDashboardConfig.tsx?

Reply:
Done

Reply:
Thanks